pages/projects.js

```diff
@@ -1,14 +1,12 @@
 import siteMetadata from '@/data/siteMetadata'
 import projectsData from '@/data/projectsData'
-import Image from '@/components/Image'
-import Link from '@/components/Link'
 import Card from '@/components/Card'
-import { PageSeo } from '@/components/SEO'
+import { PageSEO } from '@/components/SEO'
 
 export default function Projects() {
   return (
     <>
-      <PageSeo title={`Projects - ${siteMetadata.author}`} description={siteMetadata.description} />
+      <PageSEO title={`Projects - ${siteMetadata.author}`} description={siteMetadata.description} />
```

Comment:
Sorry, I just dig a bit deeper because I updated my i18n repo.

What would you think if we replaced `description={siteMetadata.description}` with `description={"Things I created"}` ? It's very very minor, but I think it could be more coherent with `tags`. `siteMetadata.description` would be left only on pages where we can access blogs posts.

Reply:
I think it is fine as it is. I changed the description of the tags page to "Things I blog about" just to add a bit of flare band hopefully nudge the user to want to modify some of the SEO tags himself.